Description: Origin field now contains template value, it should be changed as actual upstream url
Author: Hideki Yamane <henrich@debian.org>
Forwarded: Yes
Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
#Origin: <vendor|upstream|other>, <url of original patch>
--- a/foobar.h
+++ b/foobar.h
@@ -20,6 +20,6 @@
 
 
 /**
- * You may know.
+ * You may know...
  */
 